docs/BUILD.md

```diff
@@ -0,0 +1,82 @@
+# Building the App for App Store Submission
+
+First, clone the repo (requires `git` - see [Dependencies](#dependencies)):
+
+```
+git clone https://github.com/WorldHealthOrganization/app.git
+cd app
+```
+
+## iOS
+
+These instructions assume that an app record has already been added to the App Store Connect account. If you haven't done so already, you can follow [these instructions](https://help.apple.com/app-store-connect/#/dev2cd126805).
+
+### Install Dependencies
+
+`cd` into the `client/flutter/ios` directory:
+
+```
+cd client/flutter/ios
+```
+
+Install CocoaPods dependencies (requires `CocoaPods` - see [Dependencies](#dependencies)):
+
+```
+pod install
+```
+
+Open the Xcode Workspace:
+
+```
+open Runner.xcworkspace
+```
+
+### Review Xcode Build Settings
+
+For detailed procedures and descriptions, see [Prepare for app distribution](https://help.apple.com/xcode/mac/current/#/dev91fe7130a).
+
+In Xcode:
+
+1. Select **Runner** in the Xcode project navigator, then select the **Runner** target in the settings view sidebar.
+1. Select the General tab.
+1. Do not change the **Display Name** from `Runner`
+1. Verify you have chosen the same **Bundle Identifier** as the app you have registered on App Store Connect.
+1. In the **Signing** section, ensure `Automatically manage signing` is selected, and select the team associated with your Apple Developer account. If other ways of signing code are needed, review the [Code Signing Guide](https://developer.apple.com/library/archive/documentation/Security/Conceptual/CodeSigningGuide/Introduction/Introduction.html) and [Create, export, and delete signing certificates](https://help.apple.com/xcode/mac/current/#/dev154b28f09)
+
+### Create a Build Archive
+
+Configure the app version and build:
+
+1. Select **Product > Scheme > Runner**.
+1. Select **Product > Destination > Generic iOS Device**.
+1. Select **Runner** in the Xcode project navigator, then select the **Runner** target in the settings view sidebar.
+1. In the Identity section, update the **Version** number to the user-facing version number you wish to publish. For the initial App Store submission, version number should be `0.1.0`.
```

Comment:
remove reference to `0.1.0`, that version number should be shared over email and not embedded in instructions where it becomes stale